makeinfo-accept-leading-spaces by Karl Berry (taken from upstream)
---
 makeinfo/insertion.c |    5 +++++
 1 file changed, 5 insertions(+)

Index: texinfo-4.11.dfsg.1/makeinfo/insertion.c
===================================================================
--- texinfo-4.11.dfsg.1.orig/makeinfo/insertion.c	2008-02-21 20:42:26.000000000 +0100
+++ texinfo-4.11.dfsg.1/makeinfo/insertion.c	2008-02-21 20:42:44.000000000 +0100
@@ -489,6 +489,11 @@
       filling_enabled = no_indent = 0;
       inhibit_paragraph_indentation = 1;
       add_word ("START-INFO-DIR-ENTRY\n");
+      
+      /* The zsh manual, maybe others, wrongly indents the * line of the
+         direntry in the source.  Ignore that whitespace.  */
+      skip_whitespace_and_newlines ();
+      no_discard++;
       break;
 
     case documentdescription:
